core/src/test/scala/unit/kafka/server/ClientQuotaManagerTest.scala

```diff
@@ -84,12 +85,12 @@ class ClientQuotaManagerTest extends BaseClientQuotaManagerTest {
       clientQuotaManager.updateQuota(
         client1.configUser,
         client1.configClientEntity,
-        None
+        Optional.empty
       )
       clientQuotaManager.updateQuota(
         defaultConfigClient.configUser,
         defaultConfigClient.configClientEntity,
-        Some(new Quota(4000, true))
+        Optional.of(new  Quota(4000, true))
```

Comment:
There's an extra space here.
```suggestion
        Optional.of(new Quota(4000, true))
```